service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/NotificationHandler.java

```diff
@@ -318,18 +332,31 @@ private void sendNotification(String templateId, String email, Map<String, Objec
         }
     }
 
-    private void sendOrderNotificationForLocalAuthority(final CaseDetails caseDetails, final String localAuthorityCode,
-                                                        final String mostRecentUploadedDocumentUrl) {
+    private void sendGeneratedOrderNotificationToLocalAuthority(final CaseDetails caseDetails,
+                                                                final String localAuthorityCode,
+                                                                final String mostRecentUploadedDocumentUrl) {
         Map<String, Object> localAuthorityParameters =
-            orderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
+            generatedOrderEmailContentProvider.buildOrderNotificationParametersForLocalAuthority(
                 caseDetails, localAuthorityCode, mostRecentUploadedDocumentUrl);
 
         String recipientEmail = inboxLookupService.getNotificationRecipientEmail(caseDetails, localAuthorityCode);
 
-        sendNotification(ORDER_NOTIFICATION_TEMPLATE, recipientEmail, localAuthorityParameters,
+        sendNotification(ORDER_NOTIFICATION_TEMPLATE_FOR_LA, recipientEmail, localAuthorityParameters,
             Long.toString(caseDetails.getId()));
     }
 
+    private void sendOrderIssuedNotificationToHmctsAdmin(final CaseDetails caseDetails,
+                                                         final String localAuthorityCode,
+                                                         final byte[] documentContents,
+                                                         final IssuedOrderType issuedOrderType) {
```

Comment:
any value just passing in only `EventData` and `IssuedOrderType` here?

Reply:
I think this is fine, there is a similar method for LA, also documentContents is tied to the event rather than EventData so that would still need passing in separately

Reply:
hmm I think this makes sense though, you then have less params in your method